### PR TITLE
doc: kubectl explain routegroups

### LINKED ISSUE #1663: [improvement] CRD API Schema missing some field descriptions
**Is your feature request related to a problem? Please describe.**
Missing some descriptions when using `zkubectl explain` command

**Describe the solution you would like**
Being more strict with  CRD's API schema, we are missing some descriptions to some properties  (specifically the routeGroups definition so it would act as a closer and fast source of documentation)

example: 
```
$ remove-me zkubectl explain rg.spec.hosts
KIND:     RouteGroup
VERSION:  zalando.org/v1

FIELD:    hosts <[]string>

DESCRIPTION:
     <empty>
```

**Describe alternatives you've considered (optional)**
Navigating dev.zalando.net and then redirecting myself to skipper docs (which is still not explaining every single field
as an example, when backend type is `network` there is no clear way to determine that it pairs with `address` field

**Additional context (optional)**

**Would you like to work on it?**
Yes

### PR DESCRIPTION
**Pull request by szuecs**
fix #1663
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: dataclients/kubernetes/deploy/apply/routegroups_crd.yaml [hunk 1]
```diff
@@ -1,128 +1,204 @@
...
   group: zalando.org
-  version: v1
-  scope: Namespaced
   names:
-    kind: RouteGroup
-    singular: routegroup
-    plural: routegroups
     categories:
     - all
-    shortNames:
```
Comment (AlexanderYastrebov) [inline dataclients/kubernetes/deploy/apply/routegroups_crd.yaml:15]: No short name anymore?

Comment (mikkeloscar) [reply dataclients/kubernetes/deploy/apply/routegroups_crd.yaml:15]: Bug in the CRD gen specification. Fix here: https://github.com/szuecs/routegroup-client/pull/8

Comment (szuecs) [reply dataclients/kubernetes/deploy/apply/routegroups_crd.yaml:15]: Updated the pr

**Comment by szuecs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**